pact3.0/02-coin-cap.repl

```diff
@@ -0,0 +1,58 @@
+; --------------------------------------------------------------------------
+; --------------------------------------------------------------------------
+; Example of using refactoring "enforce" statements into capabilities
+
+
+; --------------------------------------------------------------------------
+; Installs account schema module
+; --------------------------------------------------------------------------
+(begin-tx)
+(env-data { "admin-keyset": ["product-dept" "dev-dept"] })
+
+(module acct-module ACCT-GOVERNANCE
+  @doc "account schema module"
+
+  (defcap ACCT-GOVERNANCE ()
+    (enforce-guard (read-keyset "admin-keyset")))
+
+  (defschema account
+    balance:integer
+    ks:keyset
+    active:bool
+  )
+)
+(commit-tx)
+
+; --------------------------------------------------------------------------
+; Installs hello-coin module with transfer and debit capabilities.
+; --------------------------------------------------------------------------
+(begin-tx)
+(env-keys ["product-dept" "dev-dept"])
+
+(module hello-coin HELLO-GOVERNANCE
+  @doc "Implementaion of hello-coin with capabilities"
+
+  (use acct-module)
+
+  (defcap HELLO-GOVERNANCE ()
+    (compose-capability (ACCT-GOVERNANCE)))
+
+  (deftable accounts:{account})
+
+  (defcap TRANSFER (from:string to:string amount:integer)
```

Comment:
`to` is unused in this capability, remove. Also maybe name it more specific to funds availability, like `CAN_TRANSFER`